composer.json

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": ">=7.2",
```

Comment:
What about restricting it to `"php": "^7.2||^8.0"` ?

Reply:
That works.

There was a thread on twitter, which I cant find now. Where PHP core devs recommended using greater than so they can mass test libs easier. If there truely isnt a rationale for restraining PHP version.

Reply:
.. it makes sense for this project :)